code/game/machinery/doors/firedoor.dm

```diff
@@ -299,7 +302,7 @@
 		return
 	..()
 	latetoggle()
-	layer = 2.6
+	layer = base_layer

```

Comment:
No it won't, because the firedoor layer is lower than the airlock layer when both are open. Firelocks get a larger layer increase when they close.

It's shown in the webm.